pom.xml

```diff
@@ -105,6 +105,13 @@
 			<version>1.10.19</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
In Devhub we also have the Annotation processor as build task:
```
			<plugin>
				<groupId>org.apache.maven.plugins</groupId>
				<artifactId>maven-compiler-plugin</artifactId>
				<version>3.0</version>
				<configuration>
					<annotationProcessors>
						<annotationProcessor>lombok.core.AnnotationProcessor</annotationProcessor>
					</annotationProcessors>
				</configuration>
			</plugin>
```

But Devhub uses a somewhat older version of Lombok so that is probably not required anymore.
http://awhitford.github.io/lombok.maven/lombok-maven-plugin/faq.html